Modules/IndividualAssessment/interfaces/AccessControl/interface.IndividualAssessmentAccessHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require_once 'Services/User/classes/class.ilObjUser.php';
+require_once 'Modules/IndividualAssessment/classes/class.ilObjIndividualAssessment.php';
+/**
+ * Mechanic regarding the access controll and roles of an objcet goes here.
+ * @author Denis Klöpfer <denis.kleofer@concepts-and-training.de>
+ */
+interface IndividualAssessmentAccessHandler {
```

Comment:
Because i thought of this as a generic interface for access and roles, that decouples the objects from rbac, though i understand that it may use the "il" since it also makes use of ilias-objects, like ilObjUser. If you prefer i may add the "il"...